core/src/androidTest/java/io/customer/sdk/data/store/AndroidManifestClientTest.kt

```diff
@@ -0,0 +1,15 @@
+package io.customer.sdk.data.store
+
+import io.customer.commontest.core.AndroidTest
+import io.customer.sdk.core.extensions.applicationMetaData
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Test
+
+class AndroidManifestClientTest : AndroidTest() {
```

Comment:
it would be great if we can test meta data merging here

Reply:
Added multiple manifests, test will not change because the changes have to be made when merging Manifest